analytics/terraform/spark-k8s-operator/addons.tf

```diff
@@ -424,6 +424,16 @@ module "eks_data_addons" {
     repository_password = data.aws_ecrpublic_authorization_token.token.password
   }
 
+  #---------------------------------------------------------------
+  # JupyterHub Add-on
+  #---------------------------------------------------------------
+  enable_jupyterhub = true
```

Comment:
Let's create a variable to make it on/off. 

`enable_jupyterhub = var.enable_jupyterhub`

Add this variable to `variables.tf` file and set it to `false` as default. We will update docs to show how to enable Jupyterhub and run the jobs